functests/config.go

```diff
@@ -0,0 +1,271 @@
+		Kind:       "ClusterRoleBinding",
+		APIVersion: "rbac.authorization.k8s.io/v1",
+	},
+	ObjectMeta: metav1.ObjectMeta{
+		Name: operatorName,
+	},
+	Subjects: []rbacv1.Subject{
+		{
+			Kind:      "ServiceAccount",
+			Name:      operatorName,
+			Namespace: operatorNamespace,
+		},
+	},
+	RoleRef: rbacv1.RoleRef{
+		Kind:     "ClusterRole",
+		Name:     operatorName,
+		APIGroup: "rbac.authorization.k8s.io",
+	},
+}
+
+var deployment = &appsv1.Deployment{
+	TypeMeta:   metav1.TypeMeta{Kind: "Deployment", APIVersion: "apps/v1"},
+	ObjectMeta: metav1.ObjectMeta{Name: operatorName},
+	Spec: appsv1.DeploymentSpec{
+		Selector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"name": operatorName,
+			},
+		},
+		Template: v1.PodTemplateSpec{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: map[string]string{
+					"name": operatorName,
+				},
+			},
+			Spec: v1.PodSpec{
+				ServiceAccountName: operatorName,
+				Containers: []v1.Container{{
+					Name:            operatorName,
+					Image:           "quay.io/cincinnati/cincinnati-operator:latest",
+					Command:         []string{operatorName},
+					ImagePullPolicy: v1.PullAlways,
+					Env: []v1.EnvVar{
+						{
+							Name:  "WATCH_NAMESPACE",
+							Value: "",
+						},
+						{
+							Name: "POD_NAME",
+							ValueFrom: &v1.EnvVarSource{
+								FieldRef: &v1.ObjectFieldSelector{
+									FieldPath: "metadata.name",
+								},
+							},
+						},
+						{
+							Name:  "OPERATOR_NAME",
+							Value: operatorName,
+						},
+						{
+							Name:  "OPERAND_IMAGE",
+							Value: "quay.io/app-sre/cincinnati:2873c6b",
+						},
+					},
+				},
+				},
+			},
+		},
+	},
+}
+
+var crd = &apiextensions.CustomResourceDefinition{
+	TypeMeta: metav1.TypeMeta{
+		APIVersion: "apiextensions.k8s.io/v1beta1",
+		Kind:       "CustomResourceDefinition",
+	},
+	ObjectMeta: metav1.ObjectMeta{
+		Name: crdName,
+	},
+	Spec: apiextensions.CustomResourceDefinitionSpec{
+		Group: "cincinnati.openshift.io",
+		Names: apiextensions.CustomResourceDefinitionNames{
+			Kind:     "Cincinnati",
+			ListKind: "CincinnatiList",
+			Plural:   "cincinnatis",
+			Singular: "cincinnati",
+		},
+		Scope:        "Namespaced",
+		Subresources: &apiextensions.CustomResourceSubresources{},
+		Validation: &apiextensions.CustomResourceValidation{
+			OpenAPIV3Schema: &apiextensions.JSONSchemaProps{
+				Properties: make(map[string]apiextensions.JSONSchemaProps),
+				Type:       "object",
+			},
+		},
+		Version: "v1beta1",
+		Versions: []apiextensions.CustomResourceDefinitionVersion{
+			{
+				Name:    "v1beta1",
+				Served:  true,
+				Storage: true,
+			},
+		},
+	},
+}
+
+var cincinnati = &cincinnativ1beta1.Cincinnati{
+	TypeMeta: metav1.TypeMeta{
+		APIVersion: "cincinnati.openshift.io/v1beta1",
+		Kind:       "Cincinnati",
+	},
+	ObjectMeta: metav1.ObjectMeta{
+		Name: customResourceName,
+	},
+	Spec: cincinnativ1beta1.CincinnatiSpec{
+		Replicas:       1,
+		Registry:       "quay.io",
```

Comment:
@rwsu Please suggest if I need to change the registry to CI registry `registry.svc.ci.openshift.org`?

Reply:
No change